helm-chart/ray-cluster/templates/raycluster-cluster.yaml

```diff
@@ -123,8 +127,12 @@ spec:
         initContainers: {{- toYaml $values.initContainers | nindent 10 }}
         {{- end }}
         containers:
-          - volumeMounts: {{- toYaml $values.volumeMounts | nindent 12 }}
-            name: ray-worker
+          - name: ray-worker
+            {{ if $values.volumeMounts }}
+            volumeMounts: {{ toYaml $values.volumeMounts | nindent 12 }}
+            {{ else }}
```

Comment:
We should not have the `else` logic. It may be hard to maintain if it relies on the configurations from another worker group.

Reply:
If we remove the `else` then `additionalWorkerGroups` do not get them as defaults.

Reply:
https://github.com/ray-project/kuberay/pull/1669#discussion_r1402699440